Lib/code.py

```diff
@@ -355,7 +355,7 @@ def interact(banner=None, readfunc=None, local=None, exitmsg=None, local_exit=Fa
         console.raw_input = readfunc
     else:
         try:
-            import readline
+            import readline  # noqa
```

Comment:
This PR adds `# noqa` comments to several `import readline` statements. `readline` is obviously a pretty special module that has an impact on code just by being imported; it doesn't need to be "used" in the same way as most modules. As a Ruff maintainer, I'd be open to us adding a special case for `readline` over at Ruff, so that it doesn't have to be manually `noqa`'d like this.

Reply:
I'm using other tools than pyflakes, I would prefer to have a generic `#noqa`. I don't think that readline is special enough to justify changing tools.

Reply:
Fair enough.